script/passenger-recycler.rb

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env ruby
+#
+# Trivial Passenger memory monitor installed via /etc/cron.hourly/. To modify
+# configuration create /etc/foreman/passenger-recycler.rb.conf file.
+#
+
+CONFIG = '/etc/foreman/passenger-recycler.rb.conf'
```

Comment:
I also think it makes sense to keep the configuration file with the same format as everything else (YAML). I know it's easy from the developer POV to just run `load(CONFIG)` but it complicates things a bit from the user POV. Also YAML files can be easily modified with our installer, unlike Ruby source..